drivers/scheduler/k8s/specs/job-pod-deployer-psa-restricted/storage.yaml

```diff
@@ -0,0 +1,12 @@
+##### Portworx persistent volume claim
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: job-pod-pvc
+spec:
+  storageClassName:  job-pod-deployer
```

Comment:
Nit: Remove the space in the beginning for name.

Reply:
Done